lib/tokens.ts

```diff
@@ -1,13 +1,13 @@
 import { unsafeCSS } from 'lit';
 
 export const mobileBreakpoint = unsafeCSS`700px`;
+export const mobilePortraitBreakpoint = unsafeCSS`320px`;
```

Comment:
Not for this PR, but because I'm looking at this change and thinking...  we should pull these in with the token project as a dependency to get these values.